azure-pipelines.yml

```diff
@@ -14,12 +14,12 @@ trigger:
 
 resources:
   repositories:
-    - repository: templates
-      type: github
-      name: ericpre/ci-scripts
-      # For more details on service connection endpoint, see
-      # https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints
-      endpoint: ericpre # Azure DevOps service connection
+  - repository: templates
+    type: github
+    name: hyperspy/ci-scripts
+    # For more details on service connection endpoint, see
+    # https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints
+    endpoint: ericpre # Azure DevOps service connection (your organisation Azure DevOps name)
```

Comment:
This doesn't need to be the the organisation name and using `ericpre` as an example is actually misleading. Maybe using `service_connection_name` would be more clear and stating that he needs to be setup on azure?

Reply:
I agree. I would just keep it as `service_connection_name`. Adding this in a new commit.